src/OSPSuite.SimModel/include/SimModel/Species.h

```diff
@@ -100,7 +100,7 @@ class Species :
 
 	void SetODEIndex(int newIndex);
 
-	virtual void AppendUsedParameters(std::set<int> & usedParameterIDs);
+	virtual void AppendUsedParameters(std::set<int> & usedParameterIDs, bool includeInitialFormula=false);
```

Comment:
What does this parameter mean `includeInitialFormula`
Sometime this extra parameter has a default value (here false) sometimes not (see FormulaChange above). Why?

Reply:
Is explained here: https://github.com/Open-Systems-Pharmacology/OSPSuite.SimModel/pull/60#pullrequestreview-200236987
The reason is: `Species.AppendUsedParameters` is called "on the top level" 
a) from the current use case and 
b) from the Model-CPP-Export). 
As I didn't want to change b), I defined this parameter as optional.

`FormulaChange.AppendUsedParameters` is called only by it's "parent" (`Switch.AppendUsedParameters`), thus here there is no need for optional paramter